bin/node/src/chain_spec.rs

```diff
@@ -109,6 +112,33 @@ impl ChainParams {
             None => "Aleph Zero Development",
         }
     }
+
+    pub fn account_ids(&self) -> Vec<AccountId> {
+        match &self.account_ids {
+            Some(ids) => ids
+                .iter()
+                .map(|id| {
+                    let id = id.as_str();
+                    let key: sr25519::Public =
+                        hex::decode(id).unwrap().as_slice().try_into().unwrap();
```

Comment:
Since this expects public keys on input and not account_ids (technically they might be the same thing, I'm not sure, but the types are different) it would be better to rename this cli argument to `authority_pkeys` or some such. I think you can also write a description of what this argument is and what format is expected, by placing a comment above the appropriate structopt.

Reply:
done